osquery/tables/system/windows/drivers.h

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+#define WIN32_LEAN_AND_MEAN
+// clang-format off
+#include <Windows.h>
+#include <initguid.h>
+// clang-format on
+#include <Devpkey.h>
```

Comment:
Do you need all of these includes in the header? The nit is we should only include the headers needed for the data structures used in the method prototypes.